src/main/java/com/hazelcast/jdbc/HazelcastJdbcStatement.java

```
@@ -0,0 +1,270 @@
+
+    }
+
+    @Override
+    public int getQueryTimeout() throws SQLException {
+        return Math.toIntExact(queryTimeout.getSeconds());
+    }
+
+    @Override
+    public void setQueryTimeout(int seconds) throws SQLException {
+        this.queryTimeout = Duration.ofSeconds(seconds);
+    }
+
+    @Override
+    public void cancel() throws SQLException {
+    }
+
+    @Override
+    public SQLWarning getWarnings() throws SQLException {
+        return null;
+    }
+
+    @Override
+    public void clearWarnings() throws SQLException {
+    }
+
+    @Override
+    public void setCursorName(String name) throws SQLException {
+    }
+
+    @Override
+    public boolean execute(String sql) throws SQLException {
+        return false;
+    }
+
+    @Override
+    public ResultSet getResultSet() throws SQLException {
+        return resultSet;
+    }
+
+    @Override
+    public int getUpdateCount() throws SQLException {
+        return 0;
+    }
+
+    @Override
+    public boolean getMoreResults() throws SQLException {
+        return false;
+    }
+
+    @Override
+    public void setFetchDirection(int direction) throws SQLException {
+
+    }
+
+    @Override
+    public int getFetchDirection() throws SQLException {
+        return 0;
+    }
+
+    @Override
+    public void setFetchSize(int rows) throws SQLException {
+        this.fetchSize = rows;
+    }
+
+    @Override
+    public int getFetchSize() throws SQLException {
+        return fetchSize;
+    }
+
+    @Override
+    public int getResultSetConcurrency() throws SQLException {
+        return 0;
+    }
+
+    @Override
+    public int getResultSetType() throws SQLException {
+        return 0;
+    }
+
+    @Override
+    public void addBatch(String sql) throws SQLException {
+
+    }
+
+    @Override
+    public void clearBatch() throws SQLException {
+
+    }
+
+    @Override
+    public int[] executeBatch() throws SQLException {
+        return new int[0];
```

Since we do not support the execution of batches for now (are we?), we should throw an exception here.